### ISSUE TITLE
Release 1.2.1 GUI fails on MacOSX

### ISSUE THREAD

**Issue by JuliaS92** [MannLabs/alphapeptdeep#217]
**Describe the bug**
Release 1.2.1 does not launch GUI on MacOSX. https://github.com/MannLabs/alphapeptdeep/releases/download/1.2.1/peptdeep_gui_installer_macos.pkg

**To Reproduce**
1. Install https://github.com/MannLabs/alphapeptdeep/releases/download/1.2.1/peptdeep_gui_installer_macos.pkg
2. run peptdeep.app

**Expected behavior**
The GUI launches

**Logs**
```
[5495] Error loading Python lib '/Applications/peptdeep.app/Contents/Frameworks/libpython3.9.dylib': dlopen: dlopen(/Applications/peptdeep.app/Contents/Frameworks/libpython3.9.dylib, 0x000A): tried: '/Applications/peptdeep.app/Contents/Frameworks/libpython3.9.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Applications/peptdeep.app/Contents/Frameworks/libpython3.9.dylib' (no such file), '/Applications/peptdeep.app/Contents/Frameworks/libpython3.9.dylib' (no such file)
```

**Version (please complete the following information):**
 - Installation Type: One-Click Installer
 - If no log is available, provide the following:
   - Platform information
     - Sonoma 14.5 on M3 Chip
   - Python information:
     - peptdeep version 1.2.1

**Comment by mschwoer** [comment]
Thanks for reporting, this has been fixed and will be released in the next couple of days.

**Comment by mschwoer** [comment]
should be fixed by release 1.3.1: https://github.com/MannLabs/alphapeptdeep/releases/tag/v1.3.1
